extensions/datasource/deployment-spi/src/main/java/io/quarkus/datasource/deployment/spi/DevServicesDatasourceContainerConfig.java

```diff
@@ -15,7 +16,7 @@ public class DevServicesDatasourceContainerConfig {
     private final Optional<String> dbName;
     private final Optional<String> username;
     private final Optional<String> password;
-    private final Optional<String> initScriptPath;
+    private final Optional<List<String>> initScriptsPaths;
```

Comment:
I'm not sure `Optional<List<...>>` makes a lot of sense, since an empty list is the same thing as no list at all...

I'd suggest `List<String>` instead of `Optional<List<String>>`?